pom.xml

```diff
@@ -252,14 +252,19 @@
     <dependency>
       <groupId>com.google.guava</groupId>
       <artifactId>guava</artifactId>
-      <version>19.0</version>
+      <version>20.0</version>
```

Comment:
you cannot update the version just to satisfy reflections. It should be a reasonable derision. Each version update should be a separate commit.